Update upgrade (#8985)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

#### Assign access to upgrade

> The <Constant name="fusion" /> upgrade assistant and readiness panel are controlled by two account-level settings that an admin must configure.

Reviewer: should we just refer to this as the Fusion readiness & upgrade flow to match the account setting?

On first reading, I thought the assistant here referred to the Fusion migration agent which could be confusing (and released at a different date with dev agent)

Step 1: Enable Fusion readiness features (required)

The upgrade assistant and readiness panel only appear after enabling this setting. From your **Account settings**:

1. Navigate to the **Account** screen.
2. Click **Edit** and scroll to the **Settings** section.
3. Click the box next to **Enable Fusion readiness & upgrade features**.
4. Click **Save**.

> Once enabled, all admins and developers can see each project's Fusion readiness status and which jobs are eligible or ineligible for <Constant name="fusion" />. Admins can also initiate the <Constant name="fusion" /> upgrade from development environments, environment settings, and job settings (subject to existing user permissions).

Reviewer: Should we mention here that admins and developers will also get debug tools to see what makes their project incompatible with Fusion and fix it? (e.g. debug in Studio, debug in Studio with Copilot, Run once on Fusion)

> Step 2: Restrict upgrade access (optional, Enterprise/Enteprise Plus only)

Reviewer 2: ```suggestion
Step 2: Restrict upgrade access (optional, Enterprise/Enterprise+ only)
```

Author: Good catch!

> By default, any user who can see the upgrade assistant can use it. To restrict upgrade execution to users with the `Fusion admin` permission set, enable this additional setting. From your **Account settings**:

Reviewer: Same comment here on "upgrade assistant"

Reviewer: We should call out again that all account admins and developers can see the readiness & upgrade flow. To restrict upgrading to a subset of these users...

> By default, all admins and developer-licensed users can access the <Constant name="fusion" /> readiness & upgrade flow. To restrict upgrade execution to users with the `Fusion admin` permission set, enable this additional setting. From your **Account settings**:

Reviewer 2: Hiya Matt

I wasn't sure where to add this comment but some references to the Fusion admin are back ticked whereas the account admin isn't? Should they both be backticked for consistency or not?

Author: They should, I'll fix that!

Author: On further review, the spots where it's isn't in a `block` is when it's a UI element, which is bolded.

1. Navigate to the **Account** screen.
2. Click **Edit** and scroll to the **Settings** section.
3. Click the box next to **Enable restricted Fusion upgrade permissions**.
4. Click **Save**.

> This hides the <Constant name="fusion" /> upgrade workflow from users who don't have the `Fusion admin` permission set. To grant access to the upgrade workflows to specific projects and/or specific users:

Reviewer 2: ```suggestion
This hides the <Constant name="fusion" /> upgrade workflows from users who don't have the [`Fusion admin`](/docs/cloud/manage-access/enterprise-permissions#fusion-admin) permission set. To grant access to the upgrade workflows to specific projects and/or specific users:
```

1. Navigate to an existing group in your **Account settings** and click **Edit**, or click [**Create group**](/docs/cloud/manage-access/about-user-access#create-new-groups) to create a new one.
2. Scroll to the **Access and permissions** section and click **Add permission**.

> 3. Select the **Fusion admin** permission set from the dropdown and then select the project(s) you want the users to access. 

Reviewer 2: ```suggestion
3. Select the **Fusion admin** permission set from the dropdown, then select the project(s) you want the users to access. 
```
